client/src/components/CardItem/index.js

```diff
@@ -0,0 +1,3 @@
+import CardItem from "./CardItem";
```

Comment:
Please, delete index.js file here and use import and reexport at components/index.js